app/views/content/train-to-be-a-teacher/school-placements.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Teacher training school placements"
+heading: "Teacher training school placements"
+subcategory: Postgraduate teacher training
+description: |-
+  Learn about teacher training school placements, including how long they are, what trainees do on placements, and the support available. 
+navigation: 20.20
+navigation_title: Teacher training school placements
+navigation_description: Learn about teacher training school placements, including how long they are, what trainees will do on placements, and the support available. 
+quote:
+  q-giorgio:
+    text: "The highlight of my placements was getting to teach!"
+    name: "Giorgio, biology teacher"
+    classes: quote--indent
+promo_content:
+    - content/train-to-be-a-teacher/promos/adviser-promo-apply
+keywords:
+    - placements
+    - school placements
+    - classroom placements
+    
+---
+
+School placements are an important and exciting part of school-led or university-led postgraduate teacher training. They're a chance to:
+
+* get experience in the classroom
+* immerse yourself in your placement schools
+* get to know your pupils and colleagues
+* start your career as a teacher
+
+## How long are placements?
+
+School placements last for a minimum of 24 weeks. This time is spread across at least 2 schools. This is to give you a varied and thorough experience in the classroom. For example, in your first placement you might teach key stage 4, and in your second placement you might teach key stage 3 in a school for pupils with [special educational needs and disabilities (SEND)](/life-as-a-teacher/age-groups-and-specialisms/special-educational-needs). 
+
+## Where will my placements take place? 
+
+Your training provider will select placement schools for you. They will contact you and discuss your situation to help them select a location that you can travel to. 
```

Comment:
Do they consider anything else? Eg if the trainee has any particular needs? Or is it just about travel?

Reply:
Worth checking with policy